ESLint / Prettier(gts) 스타일 위반 해소

**Is your feature request related to a problem? Please describe.**
gts 설정 도입 이후 `github-service.ts`, `index.ts`에서 prettier 포맷 규칙 위반이 발생하고 있다. 에디터에서 빨간 줄로 표시되며, `bun run lint` 실행 시 아래 11건의 오류가 출력된다.

```
github-service.ts
   63:28  error  Replace `labels:·{nodes:·{name:·string}[]}·|·null` with `⏎··labels:·{nodes:·{name:·string}[]}·|·null,⏎`  prettier/prettier
  125:9   error  Insert `··`                                                                                              prettier/prettier
  157:9   error  Insert `··`                                                                                              prettier/prettier
  158:1   error  Replace `······` with `········`                                                                         prettier/prettier
  182:8   error  Insert `·`                                                                                               prettier/prettier
  183:1   error  Insert `··`                                                                                              prettier/prettier
  216:9   error  Insert `··`                                                                                              prettier/prettier
  217:1   error  Insert `··`                                                                                              prettier/prettier
  220:1   error  Insert `·`                                                                                               prettier/prettier

index.ts
  109:62  error  Replace `detailed,·owner,·repoName` with `⏎············detailed,⏎············owner,⏎············repoName,⏎··········`  prettier/prettier
  115:73  error  Replace `repoData` with `⏎············repoData,⏎··········`                                                            prettier/prettier

✖ 11 problems (11 errors, 0 warnings)
```

**Specify version (commit id).**
[8c35bda](https://github.com/oss2026hnu/reposcore-ts/commit/8c35bda3cdda2e82a8e9e1cc8a9d9aad547dc5f1)

**Describe the solution you'd like**
아래 명령어로 자동 수정한 뒤 `bun run lint`가 오류 없이 통과하는지 확인한다.

```bash
bun run fix
bun run lint
```

수정 대상 파일:
- `github-service.ts` — 9건 (들여쓰기, 후행 쉼표)
- `index.ts` — 2건 (함수 인수 줄바꿈)

로직 변경 없이 포맷만 수정한다. 기존 기능을 삭제하거나 동작을 바꾸지 않는다.

**Describe alternatives you've considered**
`eslint-disable` 주석으로 오류를 억제하는 방법이 있으나, 팀 컨벤션(gts)을 우회하는 것이므로 채택하지 않는다.

**Additional context**
없음.

--- a/index.ts
+++ b/index.ts
@@ -1,17 +1,13 @@
-import { cac } from "cac";
-import pkg from "./package.json" assert { type: "json"};
+import {cac} from 'cac';
+import pkg from './package.json' assert {type: 'json'};
 
 import {createGitHubService} from './github-service';
 import {ScoreCalculator, type RepoData} from './score-calculator';
 import {summarizeRepo, writeOutputFiles} from './output';
 import type {RepoSummary} from './output';
 
-const cli = cac("reposcore-ts");
+const cli = cac('reposcore-ts');
 cli.version(pkg.version);
-
-
-
-
 
 const supportedFormats = ['csv', 'txt'] as const;
 type SupportedFormat = (typeof supportedFormats)[number];
@@ -113,13 +109,19 @@ cli
             useCache,
           );
 
-          const repoData = ScoreCalculator.calculateRepoData(detailed, owner, repoName);
+          const repoData = ScoreCalculator.calculateRepoData(
+            detailed,
+            owner,
+            repoName,
+          );
           const repoSummary = summarizeRepo(repoPath, detailed);
 
           repoDataList.push(repoData);
           repoSummaries.push(repoSummary);
 
-          const singleUserScores = ScoreCalculator.calculateUserScores([repoData]);
+          const singleUserScores = ScoreCalculator.calculateUserScores([
+            repoData,
+          ]);
           const subDir = `${owner}-${repoName}`;
           const written = await writeOutputFiles(
             format as SupportedFormat,

